src/fs.cpp

```diff
@@ -47,20 +50,38 @@ FileLock::~FileLock()
     }
 }
 
+static bool IsWSL()
```

Comment:
I just realized after merging that we could shortcut this check to `false` on non-x86 platforms. But it doesn't matter too much.